CRM/Core/Page/File.php

```diff
@@ -76,8 +76,9 @@ public function run() {
       CRM_Core_Error::statusBounce(ts('The file is either empty or you do not have permission to retrieve the file'));
     }
 
+    // FIXME: Yikes! Deleting records via GET request??
```

Comment:
But it checks for `confirmed` in the url so it's safe! 😈

Reply:
Perfectly safe. Nothing to see here, move along...